deepcell_label/templates/tool.html

```diff
@@ -76,7 +76,7 @@ <h4>{{ settings['title'] | title }}</h4>
       // disable button and show loading bar
       document.getElementById('submit').classList.add('disabled');
       document.getElementById('loading-bar').classList.add('active');
-      const upload = fetch(`${document.location.origin}/upload_file/${outputBucket}/${projectID}`,
+      const upload = fetch(`${document.location.origin}/api/upload/${outputBucket}/${projectID}`,
```

Comment:
What changed here? Did we forget to update this route in a previous PR?

Reply:
Yes, this was fixed on May 27 when Ed noticed that uploads weren't working. The deployment on caliban.deepcell.org is using this branch. The upload route changed and I forgot to update it in the submit button.